server/error/api-error.js

```diff
@@ -8,6 +8,10 @@ class ApiError {
     return new ApiError(400, msg);
   }
 
+  static unAuthenticated(msg) {
```

Comment:
I'm just being picky. Isn't "unauthenticated" a single word? lol

Reply:
Yeah! I was thinking about that too! But I liked the look of it this way lol, kinda easier to read. But either way is oK!